libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/NavigationTelemetry.java

```diff
@@ -176,8 +176,8 @@ void startSession(DirectionsRoute directionsRoute) {
     if (!isConfigurationChange) {
       navigationSessionState = navigationSessionState.toBuilder()
         .originalDirectionRoute(directionsRoute)
-        .originalRequestIdentifier(directionsRoute.routeOptions() != null
```

Comment:
So it's cool to remove this null check now?

Reply:
@ericrwolfe Yeah @cammace is working on removing the annotation in MAS, but it should never be `null`